go/libkb/sig_chain.go

```diff
@@ -155,6 +155,18 @@ func (sc SigChain) GetComputedKeyInfos() (cki *ComputedKeyInfos) {
 	return
 }
 
+func (sc SigChain) GetComputedKeyInfosWithVersionBust() (cki *ComputedKeyInfos) {
```

Comment:
The original `GetComputedKeyInfos` is still called in several places in `user.go`. Just want to make sure that that's what we want?